src/utils/unref/unref.mdx

```diff
@@ -0,0 +1,53 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="utils/unref" />
+
+# unref
+
+This module contains utilities for working with ref objects.
+
+## `Unreffable<T>`
+
+Type for a value that is optionally wrapped in a RefObject type. This is useful for cases where a
+value may be passed as either a plain value or a RefObject containing the value.
+
+### Example usage
+
+```ts
+type MyType = Unreffable<string>;
+
+const plainValue = 'foo' satisfies MyType; // plain value
+const refObjectValue = { current: 'bar' } satisfies MyType; // RefObject containing value
+```
+
+## `Unref<T extends Unreffable<unknown>>`
+
+Type to get the value of an Unreffable object. If the input type is a RefObject<T>, it returns the
+type T. Otherwise, it returns the input type itself.
+
+### Example usage
+
+```ts
```

Comment:
Either this example is incorrect, or it's extremely confusing :D
They both do `Unref<string>`, since `MyType` resolves to `string`.

Maybe something more like this is more helpful:

```ts
type TypeA = string;
type TypeB = RefObject<string>;

type A = Unref<TypeA>; // string
type B = Unref<TypeB>; // string
```